website/content/docs/nia/index.mdx

```diff
@@ -33,10 +33,12 @@ Consul-Terraform-Sync executes one or more automation tasks with the most recent
 
 -> **Note:** The terminology "tasks" used throughout the documentation refers to all types of tasks except when specifically stated otherwise.
 
-**Source Input** - A source input defines objects that provide values or metadata to the Terraform module. See [source input](/docs/nia/terraform-modules#source-input) for the supported metadata and values. For example, a user can configure a Consul KV source input to provide KV pairs as variables to their respective Terraform Module. The source input can be included in two ways. It can be specified as a parameter in a condition using `source_includes_var` and also by using the `source_input` block.
+**Source Input** - A source input defines objects that provide values or metadata to the Terraform module. See [source input](/docs/nia/terraform-modules#source-input) for the supported metadata and values. For example, a user can configure a Consul KV source input to provide KV pairs as variables to their respective Terraform module. The source input can be included in two ways. It can be specified as a parameter in a condition using `source_includes_var` and also by using the `source_input` block.
 
 **Network Infrastructure Automation (NIA)** - Enables dynamic updates to network infrastructure devices triggered when specific conditions, such as service changes and registration, are met.
 
+**Services** - A service in CTS represents a service that is registered with Consul for service discovery. Services are grouped by its service name that may have one or more instances of that service with unique ids. CTS monitors services based on service names, and can provide service instance details to a Terraform module for network automation.
```

Comment:
The `its` is throwing me off a little, I think because you have `Services`. Maybe `Services are grouped by a service name`?

Reply:
thanks for calling out `its`, I like @wilkermichael's suggestion to replace with `their`

> Services are grouped by their service names